code/game/objects/items/food/lizard.dm

```diff
@@ -551,7 +551,7 @@
 		/datum/reagent/consumable/nutriment/protein = 10,
 	)
 	tastes = list("bread" = 1, "meat" = 1)
-	foodtypes = MEAT | NUTS | RAW | GORE
+	foodtypes = MEAT | NUTS | GORE
```

Comment:
You should remove "raw" from `/obj/item/food/pizza/flatbread/rawmeat` typepath since it no longer makes sense with this change